tensorboard/webapp/app_routing/internal_utils_test.ts

```diff
@@ -21,6 +21,15 @@ import {
 } from './testing';
 import {RouteKind} from './types';
 
+function getMockReturnValuesFuntion(testUint8array: Uint8Array) {
+  return function <Uint8Array>(arr: Uint8Array): Uint8Array {
+    if (arr instanceof Uint8Array) {
+      arr.set(testUint8array);
+    }
+    return arr;
```

Comment:
You could consider throwing an Error here instead of silently doing nothing. It would make clearer that something unexpected has happened.